README-pl.md

```diff
@@ -29,17 +29,24 @@ cd website
 
 Strona Kubernetesa używa [Docsy Hugo theme](https://github.com/google/docsy#readme). Nawet jeśli planujesz uruchomić serwis w środowisku kontenerowym, zalecamy pobranie podmodułów i innych zależności za pomocą polecenia:
 
-```bash
-# pull in the Docsy submodule
+### Windows
+```powershell
+# aktualizuj podrzędne moduły
 git submodule update --init --recursive --depth 1
 ```
 
+### Linux / inne systemy Unix
+```bash
```

Comment:
do we support generic ````shell` ?

Reply:
I'm following the upstream English version. And I'm okay with this case, it's just a hint for markdown code formatting.